mk

```diff
@@ -166,6 +166,10 @@ actions['up'].add_argument(
     '-f', '--data-fraction',
     help='Use the specified fraction of the real data.',
 )
+actions['up'].add_argument(
+    '-sd', '--start-date',
```

Comment:
I'm not sure how exactly this python argument parsing works but two letter abbreviations may be confusing in the future. Usually people use a single letter only (lower or upper case).
Usually you can chain them e.g.
`script.sh -f -d -s` is equivalent to `script.sh -fds`.
I think it would be better to have a single letter abbreviations only.

Reply:
This is a fair point.  It's just hard to come up with rational for the latter - they all want to be a d or an f!  Will definitely bear in mind for the future. :+1:

Reply:
I'd go for simple `-s` or `-S` if the first one is taken. It's quite common practice. For example in `sudo` command `-h` is for help and `-H` is abbreviation for `--set-home`.
There are also a few other two letter abbreviations. `-fd` for fake data. I think that during next opportunity they should be refactored.
The sooner the better as later when people will get used to the options it's going to be harder.